doc/HOSTING

```diff
@@ -0,0 +1,116 @@
+# introduction
+
+This document tries to explain what is required to host a
+dedicated server, provided you already have a working
+unvanquished installation and "pak" files (those who have a
+`.dpk` file extension), which are not necessarily the
+official ones.
+
+# non-goals
+It does *not* tries to explain how to compile or modify
+unvanquised, how to configure a server with modified engine,
+how to setup your system or network, and aims at the
+simplest possible configuration.
+
+# pre-requisites
+
+First of all, you need a computer than can be reached by the
```

Comment:
Perhaps this might be more legible as a list rather than a block of text?
```
 - Computer with inbound access from the Internet (either a public IP or port forwarding).
 - HTTP server to host files for players to download
```
Serving on port 80 is not a requirement and may be difficult for people using port forwarding. Unvanquished will respect port numbers in the sv_downloadUrl iirc.

Reply:
Can't find this cvar when grepping the code.

Reply:
Sorry I meant `sv_wwwBaseURL`

Reply:
I simply ended up removing the port information (and writing a PR to change the sv_www cvar_t into Cvar)